### ISSUE TITLE
Intermittent failure in spec/models/user_spec.rb

### ISSUE THREAD

**Issue by jfly** [cubing/worldcubeassociation.org#479]
```
  1) User WCA ID verifies WCA ID unique when changing WCA ID
     Failure/Error: let(:user) { FactoryGirl.create :user_with_wca_id }
     ActiveRecord::RecordNotUnique:
       Mysql2::Error: Duplicate entry '2006FLEI01-1' for key 'PRIMARY': INSERT INTO `Persons` (`id`, `name`, `countryId`, `gender`, `year`, `month`, `day`) VALUES ('2006FLEI01', 'Miss Brandon Leuschke', 'USA', 'm', 1966, 4, 4)
     # ./spec/factories/users.rb:40:in `block (4 levels) in <top (required)>'
     # ./spec/models/user_spec.rb:111:in `block (3 levels) in <top (required)>'
     # ./spec/models/user_spec.rb:153:in `block (3 levels) in <top (required)>'
Finished in 46.5 seconds (files took 3.48 seconds to load)
378 examples, 1 failure
Failed examples:
rspec ./spec/models/user_spec.rb:150 # User WCA ID verifies WCA ID unique when changing WCA ID
```

**Comment by pingskills** [comment]
@jfly is this still an issue or can it be closed?

**Comment by jfly** [comment]
AFAIK, yes it is.

**Comment by jonatanklosko** [comment]
This is weird that it appears time to time but this should be fixed by #479.
